src/main/java/com/wolApi/wolApi/api/bibleVerses/BibleVerseViews.java

```diff
@@ -21,20 +21,23 @@ public class BibleVerseViews {
     public BibleVerseViews(BibleVerseService bibleVerseService) {
         this.bibleVerseService = bibleVerseService;
     }
-    @GetMapping("/get-verse/{book}/{chapter}/{verse}")
-    public Verse getVerse(@PathVariable("book") String book,
-                          @PathVariable("chapter") String chapter,
-                          @PathVariable("verse") String verse) throws IOException {
+    @GetMapping("/get-verse/{bookNum}/{chapterNum}/{verseNum}")
+    public Verse getVerse(@PathVariable("bookNum") String bookNum,
+                          @PathVariable("chapterNum") String chapterNum,
+                          @PathVariable("verseNum") String verseNum) throws IOException, InterruptedException {
```

Comment:
Use query parameters here, i.e. should look something like this: `/get-verse?bookNum={bookNum}&chapterNum={chapterNum}&verseNum={verseNum}`

Reply:
I believe the annotation is `@RequestParam`, but just double check.

Reply:
Ok so I just made a commit using request params instead of path variables -> https://github.com/j-koziel/wol-api/pull/210/commits/b7c666fe953a3cb5f5fbd8f083b134cae6a348a4